src/components/quote.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+const apiKey = 'MjFx93DKQY9OcA2CcwGaFg==97fQaXcDcvJR7aL3';
+const apiUrl = 'https://api.api-ninjas.com/v1/quotes';
+
+const Quote = () => {
+  const [data, setData] = useState({ quote: 'this is a quote', author: null });
+
+  const getQuote = async () => {
+    setData({ quote: 'Loading...', author: null });
+    try {
+      const response = await fetch(apiUrl, {
+        method: 'GET',
+        headers: { 'X-Api-Key': apiKey },
+      });
+      const responseJson = await response.json();
+      setData(responseJson[0]);
+    } catch (error) {
+      setData({ quote: 'Error fetching the Quote from API', author: null });
```

Comment:
- [ ]  Kindly also create a separate state for error and use it instead in order to make the `data` state only responsible for storing quote data.

### Hint
```javascript
const [hasError, setHasError] = useState(false)
```

Reply:
Thanks for the suggestion, I already implemented